src/applications/personalization/view-dependents/manage-dependents/schemas.js

```diff
@@ -1,8 +1,10 @@
 import { countries, states50AndDC } from 'vets-json-schema/dist/constants.json';
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import { addressUiSchema } from 'applications/vre/definitions/profileAddress';
 
 const PATTERNS = {
   date: '^(\\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)$',
```

Comment:
It seems like there should really be reusable patterns like this somewhere in a constants file in vets-website :-(